thirdweb-dev/js
# [SDK][Contracts-js][React] Use IAccountFactoryCore for detecting AccountFactory extension

## Problem solved

Use IAccountFactoryCore for detecting AccountFactory extension. This updates the SDK package to work with the latest audit fixes of smart wallet contracts, while also detecting AccountFactory extension on older account factory contracts.

## Changes made

- [ ] Public API changes: none
- [ ] Internal API changes: 
    - Use `IAccountFactoryCore` instead of `IAccountFactory` when detecting AccountFactory extension.

## How to test

- [ ] Automated tests: n/a
    - Deploy an `AccountFactory` v1.2.5 and v1.3.0 from its publish page. Both deployments should have AccountFactory extension detected on it.
